src/main/java/com/odysseusinc/athena/service/mail/EmailServiceImpl.java

```diff
@@ -141,6 +141,14 @@ public void sendVocabulariesWereSharedNotification(AthenaUser recipient, AthenaU
                 .map(AthenaUser::getEmail).toArray(size -> new String[size]);
     }
 
+    private Map<String, Object> buildParameters(String url, CDMVersion version, String vocabularyReleaseVersion, String bundleName, Map<String, String> requestedVocabularies) {
+
+        final Map<String, Object> parameters = buildParameters(url, version, vocabularyReleaseVersion);
```

Comment:
I don't see anywhere usage  final keyword with variable definition, can you align it with the rest of code

Reply:
I am using IntelliJ hotkey for variable extraction which put final keyword automatically. It is good because it enforces immutability design. Maybe we can leave as it is default and overally the right way?

Reply:
I don't mind of final, I just meant that new code should be aligned with existed, so remove final here or put it for every variable

Reply:
yep, I've add more final